app/src/main/res/values-zh-rCN/strings.xml

```diff
@@ -70,6 +70,17 @@
          needs to request permission to access the device's external storage such as the
          sdcard. '%1$s' will be replaced at runtime with the website's domain name. -->
     <string name="permission_persistent_storage">您是否要允许 %1$s 持久存储数据？</string>
+
```

Comment:
AFAIK translations must be added in Weblate directly otherwise the tool will mess everything up. So we should not include this here.

Reply:
I have removed them from this PR.